src/Riok.Mapperly/Emit/SourceEmitter.cs

```diff
@@ -14,7 +14,7 @@ public static class SourceEmitter
     public static CompilationUnitSyntax Build(MapperDescriptor descriptor, CancellationToken cancellationToken)
     {
         var ctx = new SourceEmitterContext(
-            descriptor.Symbol.IsStatic,
+            descriptor.Symbol.IsStatic || descriptor.Static,
```

Comment:
The first part is not needed, is it? Could be simplified to `descriptor.Static`.